Source/NSKeyedArchiver.m

```diff
@@ -460,6 +460,26 @@ + (NSData*) archivedDataWithRootObject: (id)anObject
   return AUTORELEASE(d);
 }
 
++ (NSData *) archivedDataWithRootObject: (id)anObject
```

Comment:
I think this method should not call archivedDataWithRootObject: instead it should be the other way around. This is the more general method and it should just have the code from archivedDataWithRootObject: and set the flag requiresSecureCoding on the created NSKeyedArchiver. As this flag is also still missing there is not much use in adding this method.
There is also no reason to return an error when secure coding is requested.

Reply:
Done